docs/src/manual/ansatz/mps.md

```diff
@@ -1,34 +1,121 @@
 # Matrix Product States (MPS)
 
-Matrix Product States (MPS) are a Quantum Tensor Network ansatz whose tensors are laid out in a 1D chain.
-Due to this, these networks are also known as _Tensor Trains_ in other mathematical fields.
-Depending on the boundary conditions, the chains can be open or closed (i.e. periodic boundary conditions).
-
-```@setup viz
+```@setup plot
+using Tenet
 using Makie
 Makie.inline!(true)
 set_theme!(resolution=(800,200))
 
 using CairoMakie
 using GraphMakie
+CairoMakie.activate!(type = "svg")
 
-using Tenet
 using NetworkLayout
 ```
 
-```@example viz
-tn = rand(MPS; n=10, maxdim=4) # hide
-graphplot(tn, layout=Spring(iterations=1000, C=0.5, seed=100)) # hide
+Matrix Product States ([`MPS`](@ref)) are a Quantum Tensor Network [`Ansatz`](@ref) whose tensors are laid out in a 1D chain.
+Due to this, these networks are also known as _Tensor Trains_ in other scientific fields.
+Depending on the boundary conditions, these chains can be open or closed (i.e. periodic boundary conditions). Currently
+only `Open` boundary conditions are supported in `Tenet`.
+
+```@example plot
+fig = Figure() # hide
+open_mps = rand(MPS; n=10, maxdim=4)
+
+plot!(fig[1,1], open_mps, layout=Spring(iterations=1000, C=0.5, seed=100)) # hide
+Label(fig[1,1, Bottom()], "Open") # hide
+
+fig # hide
+```
+
+The default ordering of the indices on the `MPS` constructor is (physical, left, right), but you can specify the ordering by passing the `order` keyword argument:
+
+```@repl plot
+mps = MPS([rand(4, 2), rand(4, 8, 2), rand(8, 2)]; order=[:l, :r, :o])
+```
+where `:l`, `:r`, and `:o` represent the left, right, and outer physical indices, respectively.
+
+
+### Canonical Forms
+
+An `MPS` representation is not unique: a single `MPS` can be represented in different canonical forms. The choice of canonical form can affect the efficiency and stability of algorithms used to manipulate the `MPS`.
+The current form of the `MPS` is stored as the trait [`Form`](@ref) and can be accessed via the `form` function:
+
+```@repl plot
+mps = MPS([rand(2, 2), rand(2, 2, 2), rand(2, 2)])
+
+form(mps)
+```
+> :warning: Depending on the form, `Tenet` will dispatch under the hood the appropriate algorithm which assumes full use of the canonical form, so be careful when making modifications that might alter the canonical form without changing the trait.
+
+`Tenet` has the internal function [`Tenet.check_form`](@ref) to check if the `MPS` is in the correct canonical form. This function can be used to ensure that the `MPS` is in the correct form before performing any operation that requires it.
```

Comment:
I've been referring Tenet without the code formatting... I think we should keep it like normal text format.

```suggestion
Tenet has the internal function [`Tenet.check_form`](@ref) to check if the `MPS` is in the correct canonical form. This function can be used to ensure that the `MPS` is in the correct form before performing any operation that requires it.
```